### PR TITLE
Loosen htmlpurifier version

### PR DESCRIPTION
**Pull request by machour**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #20080

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -73,7 +73,7 @@
         "ext-ctype": "*",
         "lib-pcre": "*",
         "yiisoft/yii2-composer": "~2.0.4",
-        "ezyang/htmlpurifier": "^4.6",
+        "ezyang/htmlpurifier": ">=4.6",
```
Comment (schmunk42) [inline composer.json:76]: This is an unbound version constraint and should be avoided.
Which version would you need? 

`^4.6` would install `4.17` btw: https://semver.madewithlove.com/?package=ezyang%2Fhtmlpurifier&constraint=%5E4.6

**Review by schmunk42 — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (machour) [reply composer.json:76]: I need 4.7 to get installed (and a coffee obviously)

**Comment by machour** [comment]
Sorry for the noise, my troubles were caused by phpoffice/phpspreadsheet

**Comment by terabytesoftw** [comment]
> Sorry for the noise, my troubles were caused by phpoffice/phpspreadsheet

No problem, nice to hear from you.

**Pull request closed without merging**